lib/qesapdeployment.pm

```diff
@@ -2207,22 +2206,6 @@ sub qesap_az_clean_old_peerings {
     }
 }
 
-=head2 qesap_az_get_native_fencing_type
-
-    Gets the native fencing type (spn/msi)
-
-=cut
-
-sub qesap_az_get_native_fencing_type {
```

Comment:
I decided to delete this method as, looking at the code, it is all about getting settings and validate them.

Validation of content is already present in other places.

This method is called only once, so I decided to just call `get_var` there instead of using a method